packages/expo-splash-screen/README.md

```diff
@@ -24,17 +24,39 @@
 
 Scale the image uniformly (maintaining the image's aspect ratio) so that both dimensions the width and height of the image will be equal to or less than the corresponding dimension of the device's screen.
 
-| Android                                                   | iOS                                                        |
-|-----------------------------------------------------------|------------------------------------------------------------|
-| <img src="./assets/demo-android-contain.gif" height="350" /> | <img src="./assets/demo-ios-contain.gif" height="350" />  |
+<table>
+  <thead><tr><td>Android</td><td>iOS</td></tr></thead>
+  <tbody><tr>
+<td>
+
+https://user-images.githubusercontent.com/379606/120575867-aeeb3580-c3d6-11eb-825d-19a847fe30f5.mp4
```

Comment:
does this work outside of github? this readme also lives on npm

Reply:
It works only on GitHub, which needs this exact syntax. Markdown otherwise doesn't support video so there's not a lot we can do here. I am not sure we can even show an image preview just on npm while letting GitHub show videos. This is what it looks like in a non-GH Markdown view:

<img width="976" alt="readme" src="https://user-images.githubusercontent.com/379606/123365275-d000fb00-d52a-11eb-82f5-babecc6f1aa7.png">